src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2148,6 +2148,21 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     info[jss::validation_quorum] = static_cast<Json::UInt>(
         app_.validators ().quorum ());
 
+    if (auto when = app_.validators().expires())
```

Comment:
I'm wondering if this should be admin-only.
However it does seem to be in a similar category as `validation_quorum`, which is currently not admin-only.

Reply:
I could be convinced otherwise, but I like matching `validation_quorum`, so that if you ever do get stale and `validation_quorum` goes to "infinity", you would know when it happened.